py-rse-index.Rmd

```diff
@@ -72,100 +125,46 @@ Sami Virtanen
 
 ### Prerequisites
 
-Learners must be able to:
+Learners must already be able to:
 
--   Write reusable functions to analyze tabular data using [R][r] or [Python][python].
+-   Write functions to analyze tabular data using [Python][python].
+    (A companion volume covers this material using [R][r].)
 -   Keep track of their work using [Git][git] and [GitHub][github].
--   Create reproducible reports using [R Markdown][r-markdown] or [Jupytext][jupytext] (in [JupyterLab Notebooks][jupyter]).
--   Publish those reports with appropriate metadata such as a [DOI][doi] and [ORCID][orcid].
+-   Create reproducible reports using [JupyterLab Notebooks][jupyter].
+-   Publish reports with appropriate metadata such as a [DOI][doi] and [ORCID][orcid].
```

Comment:
This can be removed, because DOIs and ORCIDs are covered in Chapter 16.

Reply:
good catch - taken out